experiment/autobumper/bumper/bumper.go

```diff
@@ -84,55 +84,55 @@ func (af *fileArrayFlag) Set(value string) error {
 	UpstreamURLBase string `yaml:"upstreamURLBase"`
-	//The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
+	// The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
 	IncludedConfigPaths []string `yaml:"includedConfigPaths"`
-	//The config paths to be excluded in this bump, in which only .yaml files will be considered.
+	// The config paths to be excluded in this bump, in which only .yaml files will be considered.
 	ExcludedConfigPaths []string `yaml:"excludedConfigPaths"`
-	//The extra non-yaml file to be considered in this bump.
+	// The extra non-yaml file to be considered in this bump.
 	ExtraFiles []string `yaml:"extraFiles"`
-	//The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
+	// The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
 	TargetVersion string `yaml:"targetVersion"`
-	//The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
+	// The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
 	RemoteName string `yaml:"remoteName"`
-	//The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
+	// The name of the branch that will be used when creating the pull request. If unset, defaults to "autobump".
 	HeadBranchName string `yaml:"headBranchName"`
-	//List of prefixes that the autobumped is looking for, and other information needed to bump them
+	// List of prefixes that the autobumped is looking for, and other information needed to bump them. Must have at least 1 prefix.
 	Prefixes []Prefix `yaml:"prefixes"`
 }
 
 // Prefix is the information needed for each prefix being bumped.
 type Prefix struct {
-	//Name of the tool being bumped
+	// Name of the tool being bumped
 	Name string `yaml:"name"`
-	//The image prefix that the autobumper should look for
+	// The image prefix that the autobumper should look for
 	Prefix string `yaml:"prefix"`
-	//File that is looked at when bumping to match upstream
+	// File that is looked at to determine current upstream image when bumping to upstream. Required only if targetVersion is "upstream"
 	RefConfigFile string `yaml:"refConfigFile"`
-	//File that is looked at when bumping to match upstreamStaging
+	// File that is looked at to determine current upstream staging image when bumping to upstream staging. Required only if targetVersion is "upstream-staging"
 	StagingRefConfigFile string `yaml:"stagingRefConfigFile"`
-	//Repo used when generating pull request
+	// Repo that the PR for the bump will be pushed to. This is used to generate comparison link in the PR summary.
```

Comment:
What? Those are two different repos for everyone else: One repo the PR gets created against, one for the diffing

Reply:
Yeah, I tried to clarify the distinction between the different repos at play in my review comment. I think we're talking about at least 3 repos in total if we include the Bot's fork. We could even be talking about 4 if the reference file used to choose the next version to bump to is in another repo.

Reply:
Sorry about the misunderstanding. I was confused as to exactly how this was used and I was looking at an example where the source and target repo were the same. This refers to the source repo